README.md

```diff
@@ -22,7 +22,7 @@ Learn about our [design principles](/docs/developer/design-principles.md) and [a
 
 We publish NGINX Kubernetes Gateway releases on GitHub. See our [releases page](https://github.com/nginxinc/nginx-kubernetes-gateway/releases).
 
-The latest release is [0.3.0](https://github.com/nginxinc/kubernetes-ingress/releases/tag/v0.3.0).
+The latest release is [0.4.0](https://github.com/nginxinc/kubernetes-ingress/releases/tag/v0.4.0).
```

Comment:
The changes in this file will also have to be updated in main as well.

Reply:
Yep, it's called out in the [release doc](https://github.com/nginxinc/nginx-kubernetes-gateway/blob/main/docs/release-process.md). I'll wait to open that PR until after this is merged though, I figure the CHANGELOG may have some feedback.